doc/Comments.xml

```diff
@@ -343,6 +343,11 @@ use @ExampleSession:
 ]]></Listing>
 It inserts an example into the manual just as @Example would do, but all lines are commented
 and therefore not executed when the file is read.
+All lines have to start with an &AutoDoc; comment (<C>#!</C>).
```

Comment:
Hrm, what does "All lines" here mean? Clearly in the above example, not all lines start with `#!`